flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -61,25 +61,108 @@ object AggregateUtil {
    * }}}
    *
    */
-  def createOperatorFunctionsForAggregates(
+    def createOperatorFunctionsForAggregates(
```

Comment:
I had thought about calling transformToAggregateFunctions explicitly before I had thought about calling transformToAggregateFunctions explicitly before createOperatorFunctionsForAggregates in DataSetAggregate. But in DataSetAggregate will not use transformToAggregateFunctions's return results In addition to calling createOperatorFunctionsForAggregates. so,i hope overload createOperatorFunctionsForAggregates. What do you think @fhueske ?